src/hooks/VoiceRecorder/useVoiceRecorder.tsx

```diff
@@ -73,7 +73,7 @@ export const useVoiceRecorder = ({
 
   // Timer
   const [recordingTime, setRecordingTime] = useState<number>(0);
-  let timer: ReturnType<typeof setInterval> = null;
+  let timer: ReturnType<typeof setInterval> | null = null;
```

Comment:
This isn't related to this PR, but it seems like this code should be updated with `useRef`
  
```ts
const timer = useRef<ReturnType<typeof setInterval> | null>(null);

// ...
timer.current = setInterval;

// ...
clearInterval(timer.current);
timer.current = null;
```